modules/transport-netty4/src/test/java/org/elasticsearch/http/netty4/Netty4HttpClient.java

```diff
@@ -84,7 +85,10 @@
     private final Bootstrap clientBootstrap;
 
     Netty4HttpClient() {
-        clientBootstrap = new Bootstrap().channel(NioSocketChannel.class).group(new NioEventLoopGroup());
+        clientBootstrap = new Bootstrap()
```

Comment:
Could we maybe not change this or make it use a single threaded `NioEventLoopGroup` until we've successfully dealt with https://github.com/elastic/elasticsearch/issues/46803 (just reopened, the fact that every thread we touch is using 1MB of direct memory, weakly-referenced as a `ThreadLocal` with the copy bytes channel is causing trouble in CI in some tests and this seems like a spot where we'd add more trouble potentially) in full?